code/modules/modular_armor/attachables.dm

```diff
@@ -185,3 +185,22 @@
 	parent.soft_armor = parent.soft_armor.detachArmor(soft_armor)
 	parent.slowdown -= slowdown
 	return ..()
+
+/** Extra armor module */
+/obj/item/armor_module/attachable/ballistic_armor
+	name = "\improper Ballistic Armor Reinforcement"
+	desc = "Designed for mounting on the Jaeger Combat Exoskeleton. A substantial amount of additional armor plating designed to fit inside some of the vulnerable portions of the Jaeger Combat Exoskeleton conventional armor patterns against bullets and nothing else. Will definitely impact mobility."
+	icon_state = "mod_ff_icon"
+	item_state = "mod_ff"
+	soft_armor = list("melee" = 0, "bullet" = 40, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	slowdown = 0.2
```

Comment:
i'd give it 0.1 slowdown myself, seeing its a module that makes your team not fucking bork you down, but this is fine and if it is not good we can buff/nerf it